docker-compose.yml

```diff
@@ -10,7 +10,7 @@ sut:
     - bitcoind-username-only
 
 bitcoind:
-  image: seegno/bitcoind:0.13-alpine
+  image: seegno/bitcoind:0.13.2-alpine
```

Comment:
Did you add support for the newer methods that this releases brings? Otherwise I'd prefer pinning to the older version (0.13.0?) and adding a PR to add support for 0.13.2.